README.md

```diff
@@ -0,0 +1,16 @@
+# TGA, RGB and CRY tools
+A set of tools originaly developed for PC/MSDOS.<br>
+Adapted for Visual Studio 2019 64 bits.
```

Comment:
"Adapted for Linux (32/64-bit x86)  and Visual Studio 2019 (64-bit x86)"

Reply:
Replaced with your line.